skbio/stats/ordination/_ordination_results.py

```diff
@@ -399,3 +399,44 @@ def _format_attribute(self, attr, attr_label, formatter):
         else:
             formatted_attr = formatter(attr)
         return "\t%s: %s" % (attr_label, formatted_attr)
+
+    def rename(self, mapper, axis='samples', strict=True):
+        """
+        Renames ids in OrdinationResults.
+
+        Parameters
+        ----------
+        mapper : dict or function
+            A dictionary or function that maps current ids to new ids.
+        axis : samples or features
+            Specifies which id type, either 'samples' or 'features', to rename.
+        strict: bool, optional
+           If True, then every id in the OrdinationResults dataframe must be
+           included in rename_dict and renamed.
+           If False, then only the specified indices will be renamed.
+
+        Raises
+        ______
+        ValueError
+            If strict is True and rename_dict does not contain all of the same
+            ids as OrdinationResults.
+
+        """
+        if axis == 'samples':
```

Comment:
The member method of `OrdinationResults` is `samples` so it's self consistent. I don't have strong feelings though, and this is not a API breaking risk either

Reply:
I see. This makes sense. I am thinking that it probably shouldn't even be called `axis` because samples and features are separate data frames in the ordination result. It may even be confusing because the user may think of principal components when seeing the word `axis`. What about `matrix` instead?